src/TesterInternal/TimerTests/ReminderTests_AzureTable.cs

```diff
@@ -300,12 +287,12 @@ public void TestCleanup()
             Task<bool> t1 = Task.Factory.StartNew(() =>
             {
 
-                StopSilo(siloToKill);
+                siloHost.StopSilo(siloToKill);
```

Comment:
1) yes, making this a property and pascal casing it was one of the "polishing" tasks that I had in my TODO list.
2) I like the idea of calling it something else, I just used SiloHost as a shorthand for TestingSiloHost which is the current class name we are using. I like HostedCluster better as a property name (but are you suggesting also renaming the class? that's a bigger breaking change, not sure how many folks are using it). Nevertheless, it's technically more than that, as it abstract both the cluster and the client in it.
3) Sure, I think it can evolve to account for both:
 - multi-cluster: although not sure how many tests will need that, apart from the tests that actually assert that the multi-cluster feature works. I mean, I don't see to many Orleans consumers using it, but I might be wrong.
 - Client isolated in it's app-domain: this could be very useful to be able to parallelize many fixtures, as they could all start their testing cluster with randomized ports, and also their clients in a separate app domain. This work is not trivial (Sebastian is doing it, but it requires some massaging in the way you write the tests), so that's why I didn't do it as part of this PR. I definitely would like to see it evolve in that direction, and rely less and less on statics. Even better, if I could make Orleans itself not rely on statics, even better, no need for a separate AppDomain for the clients